sdl/struct_test.go

```diff
@@ -84,6 +83,7 @@ func TestStructABI(t *testing.T) {
 		{ControllerAxisEvent{}, cControllerAxisEvent{}, nil},
 		{ControllerButtonEvent{}, cControllerButtonEvent{}, nil},
 		{ControllerDeviceEvent{}, cControllerDeviceEvent{}, nil},
+		{AudioDeviceEvent{}, cAudioDeviceEvent{}, nil},
```

Comment:
This test needs some sort of SDL >=2.0.4 check. I'm not sure how to do it.

Reply:
Hi @malashin, what do you mean by SDL >= 2.0.4 check? It seems to work fine on SDL 2.0.0.

Reply:
If it works in < 2.0.4 then it means that the stub struct for SDL_AudioDeviceEvent is being compared in the test and there is no need for another check in the tests themselves.

Everything is fine then.

I tested the new event and don't get anything on removal of an audio device (not even a CommonEvent), but that must be a problem in SDL itself. AUDIODEVICEADDED works fine.

Reply:
I think SDL_AUDIODEVICEREMOVED is emitted only when you have opened the audio device but seems like it might work on different level (e.g. simply unplugging audio jack might not be enough). I'll try on my USB audio device at home later.

Anyway the PR looks good to me so I'm merging it. Thanks!